plugins/utilities/icons_keyboard.py

```diff
@@ -20,15 +21,29 @@
     list_of_icons.append('‎')
 
 
-# ba_meta export keyboard
-class IconKeyboard(babase.Keyboard):
-    """Keyboard go brrrrrrr"""
-    name = 'Icons by \ue048Freaku'
-    chars = [(list_of_icons[0:10]),
-             (list_of_icons[10:19]),
-             (list_of_icons[19:26])]
-    nums = ['‎' for i in range(26)]
-    pages = {
-        f'icon{i//26+1}': tuple(list_of_icons[i:i+26])
-        for i in range(26, len(list_of_icons), 26)
-    }
+if hasattr(bauiv1, 'Keyboard'):
+    # ba_meta export bauiv1.Keyboard
+    class IconKeyboard(bauiv1.Keyboard):
+        """Keyboard go brrrrrrr"""
+        name = 'Icons by \ue048Freaku'
+        chars = [(list_of_icons[0:10]),
+                 (list_of_icons[10:19]),
+                 (list_of_icons[19:26])]
+        nums = ['‎' for i in range(26)]
+        pages = {
+            f'icon{i//26+1}': tuple(list_of_icons[i:i+26])
+            for i in range(26, len(list_of_icons), 26)
+        }
+else:
+    # ba_meta export keyboard
+    class IconKeyboard(babase.Keyboard):
+        """Keyboard go brrrrrrr"""
+        name = 'Icons by \ue048Freaku'
+        chars = [(list_of_icons[0:10]),
+                 (list_of_icons[10:19]),
+                 (list_of_icons[19:26])]
+        nums = ['‎' for i in range(26)]
+        pages = {
+            f'icon{i//26+1}': tuple(list_of_icons[i:i+26])
+            for i in range(26, len(list_of_icons), 26)
+        }
```

Comment:
afaik game parses `# ba_meta export` lines before interpreting the python plugin code; so based on my understanding of this so far, having `# ba_meta export` under if-else shouldn't be of help here?

@Freaku17 what happens if you remove if-else here and rename the 2nd export class as `class IconKeyboardLegacy` something?

---------------------

@Loup-Garou911XD @imayushsaini @Dliwk @SenjuZoro or anyone else willing to experiment with what might be happening here?

Reply:
I already experimented & did what I could

Reply:
I will investigate this tomorrow